src/electron/office/office_client.cc

```diff
@@ -176,6 +177,9 @@ std::string OfficeClient::GetLastError() {
 
 v8::Local<v8::Value> OfficeClient::LoadDocument(v8::Isolate* isolate,
                                                 const std::string& path) {
+   office_->setOptionalFeatures(
+      LibreOfficeKitOptionalFeatures::LOK_FEATURE_NO_TILED_ANNOTATIONS);
```

Comment:
This option does not affect the document boundaries, this is for tracked changes.

To disable the boundaries (the little gray L-shaped marks, it's a preference).

Reply:
Further, this is just one of the pre-reqs for actually getting comments as JSON and receiving callbacks, etc.